acl.yaml

```diff
@@ -1,25 +1,33 @@
 package:
   name: acl
   version: 2.3.2
-  epoch: 1
+  epoch: 2
   description: "access control list utilities"
   copyright:
     - license: LGPL-2.1-or-later AND GPL-2.0-or-later
 
 environment:
   contents:
     packages:
+      - autoconf
+      - automake
       - attr-dev
       - build-base
       - busybox
       - ca-certificates-bundle
+      - libtool
+      - gettext-dev
```

Comment:
I don't know, but wonder if this is causing the build failure, are both these needed now?

Reply:
Now that we have to call the `autogen.sh` script, we require a tool called `autopoint` which is included in gettext-dev, and needed `aclocal` which is in `automake`. The other make commands needed additional context too when I was running `make package/acl`, specifically libtool.